shotover/src/transforms/kafka/sink_cluster/scram_over_mtls.rs

```diff
@@ -123,6 +123,9 @@ async fn task(
                 let passed = instant.elapsed();
                 tracing::info!("Delegation token for {username:?} recreated in {passed:?}");
                 token_creation_time_metric.record(passed);
+
+                // TODO: We would expire the old token here if it were possible, but it is not since kafka will not allow users (even super users) to expire a token belonging to another user.
```

Comment:
I think its worth adding a second line: "However, at this point, the token will automatically expire itself in `delegation_token_lifetime / 2`, so it is not a huge concern."
Otherwise it might seem like the token would never expire.